### ISSUE TITLE
MUI/Material and Icons causes Codesandbox to load slowly or hang

### ISSUE THREAD

**Issue by mx-vs** [codesandbox/codesandbox-client#6571]
# 🐛 bug report

## Preflight Checklist

- [x] I have read the [Contributing Guidelines][contributing] for this project.
- [x] I agree to follow the [Code of Conduct][code_of_conduct] that this project
      adheres to.
- [x] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem
In a fresh project adding material and material icons causes Codesandbox to get stuck on "Transpiling Modules" or white screen or display the following image:
![qweqqe](https://user-images.githubusercontent.com/61378807/159321852-9aff5603-dba7-4974-b4fc-45f95403b050.PNG)

## How has this issue affected you? What are you trying to accomplish?
I cannot continue the project while this issue is present. 

### To Reproduce
Add @mui/material @emotion/react @emotion/styled and @mui/icons-material.

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | latest  |
| Browser          |  Chrome  |
| Operating System |    Win 10  |

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

**Comment by JamesACS** [comment]
Hey @mx-vs - I've just loaded this on our end without issue. Did you manage to solve this?

**Comment by mx-vs** [comment]
> Hey @mx-vs - I've just loaded this on our end without issue. Did you manage to solve this? 

Hey James, no unfortunately this still persists. I have tried creating completely new project but it is still the same.

**Comment by mx-vs** [comment]
> > @mx-vs - This would mean it's likely a local issue. Are you running any adblockers or extensions? 
> 
> Oh didn't think of that. I disabled all extensions and now it seems to work fine. No issues. Thank you!

Scratch that, it started to showing the error again when I tried adding a color property to a Link element from material ui.

**Comment by JamesACS** [comment]
@mx-vs - Is that change saved to the Sandbox? It's still working on this end!

**Comment by mx-vs** [comment]
> @mx-vs - Is that change saved to the Sandbox? It's still working on this end! 

Yep it is saved. But I noticed something new. It started showing the same error when I tried to create a new react component and import it. I am a bit lost now. I cleared Chrome cache and restarted but again the same.

**Comment by mx-vs** [comment]
> > @mx-vs - Is that change saved to the Sandbox? It's still working on this end! 
> 
> Yep it is saved. But I noticed something new. It started showing the same error when I tried to create a new react component and import it. I am a bit lost now. I cleared Chrome cache and restarted but again the same.

I changed the import of mui to the old one - @material-ui/core with version 4.12.3 and it seems to not show the issue anymore.
